spec/models/tax_tribunal_decision_spec.rb

```diff
@@ -0,0 +1,177 @@
+      },
+      "routes" => [
+        {
+          "path" => "/tax-and-chancery-tribunal-decisions/example-tax-tribunal-decision-#{n}",
+          "type" => "exact",
+        }
+      ],
+      "redirects" => [],
+      "update_type" => "major",
+    }
+  end
+
+  let(:non_tax_tribunal_decision_content_item) {
+    {
+      "content_id" => SecureRandom.uuid,
+      "base_path" => "/other-documents/not-a-tax-tribunal-decision",
+      "format" => "specialist_document",
+      "details" => {
+        "metadata" => {
+          "document_type" => "not_a_tax_tribunal_decision",
+        },
+      },
+    }
+  }
+
+  let(:tax_tribunal_decision_org_content_item) {
+    {
+      "base_path" => "/courts-tribunals/upper-tribunal-tax-and-chancery-chamber",
+      "content_id" => "1a68b2cc-eb52-4528-8989-429f710da00f",
+      "title" => "Upper Tribunal (Tax and Chancery Chamber)",
+      "format" => "placeholder_organisation",
+      "need_ids" => [],
+      "locale" => "en",
+      "updated_at" => "2015-08-20T10:26:56.082Z",
+      "public_updated_at" => "2015-04-15T10:04:28.000+00:00",
+      "phase" => "live",
+      "analytics_identifier" => "CO1133",
+      "links" => {
+        "available_translations" => [
+          {
+
+            "title" => "Upper Tribunal (Tax and Chancery Chamber)",
+            "base_path" => "/courts-tribunals/upper-tribunal-tax-and-chancery-chamber",
+            "description" => nil,
+            "api_url" => "https://www-origin.integration.publishing.service.gov.uk/api/content/courts-tribunals/upper-tribunal-tax-and-chancery-chamber",
+            "web_url" => "https://www-origin.integration.publishing.service.gov.uk/courts-tribunals/upper-tribunal-tax-and-chancery-chamber",
+            "locale" => "en",
+            "analytics_identifier" => "PB1140",
+          }
+        ]
+      },
+      "description" => nil,
+      "details" => {},
+    }
+  }
+
+  let(:indexable_attributes) {
+    {
+      "title" => "Example Tax Tribunal Decision 0",
+      "description" => "This is the summary of example Tax Tribunal Decision 0",
+      "link" => "/tax-and-chancery-tribunal-decisions/example-tax-tribunal-decision-0",
+      "indexable_content" => "## Header" + ("\r\n\r\nThis is the long body of an example Tax Tribunal Decision" * 10),
+      "public_timestamp" => "2015-11-16T11:53:30+00:00",
+      "tribunal_decision_category" => "banking",
+      "tribunal_decision_decision_date" => "2015-07-30",
+      "organisations" => ["upper-tribunal-tax-and-chancery-chamber"]
+    }
+  }
+
+  let(:fields) { %i[base_path content_id] }
+
+  let(:tax_tribunal_decisions) { 10.times.map { |n| tax_tribunal_decision_content_item(n) } }
+
+  before do
+    publishing_api_has_fields_for_format('specialist_document', tax_tribunal_decisions, fields)
+
+    tax_tribunal_decisions.each do |decision|
+      publishing_api_has_item(decision)
+    end
+
+    Timecop.freeze(Time.parse("2015-12-18 10:12:26 UTC"))
+  end
+
+  describe "#all" do
```

Comment:
I usually do `.` for class methods and `#` for instance methods. Happy to change if it's inconsistent...